leaf-ui/index.html

```diff
@@ -73,6 +77,8 @@
 
             <button id="symbolic-btn" class="blue-btn btn">Model Constraints</button>
             <button id="analysis-btn" class="green-btn btn">Analysis</button>
+            <button id="cycledetect-btn" class="pink-btn btn">Cycle Checker</button>
```

Comment:
Sorry, I should have specified this.
Can we change the name of this button to "Detect Cycles"